x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_alerts/alert_tags.cy.ts

```diff
@@ -66,13 +71,13 @@ describe('Alert tagging', { tags: ['@ess', '@serverless'] }, () => {
     updateAlertTags();
     cy.get(ALERTS_TABLE_ROW_LOADER).should('not.exist');
     // Then add tags to both alerts
-    selectNumberOfAlerts(2);
+    selectNumberOfAlerts(5);
```

Comment:
Why here, do we need load `auditbeat_multiple` ?

Reply:
@nkhristinin we might not need to load it, but my approach was to not want to change the logic of the test file itself. The test was previously loading the `auditbeat` file that contained 2 documents. Some tests (like this very one) were selecting multiple alerts, so I figured that to be on the safe side I would keep that logic.